app/scripts/util/data/otp.js

```diff
@@ -25,7 +25,6 @@ const Otp = function(url, params) {
     this.url = url;
 
     this.type = params.type;
-    this.issuer = params.issuer;
```

Comment:
but why?

Reply:
Was that intentional? There were two lines of `this.issuer = params.issuer;`

Reply:
I see, it's assigned below, thanks!